### ISSUE TITLE
Perhaps remove crypto.stackexchange.com URL from source code - flagged as suspicious by VaultPress

### ISSUE THREAD

**Issue by slaFFik** [phpseclib/phpseclib#1481]
This link
https://github.com/phpseclib/phpseclib/blob/7e72d923ceb8b7456a64149f10d6b04c43e9281a/phpseclib/Math/BigInteger/Engines/Engine.php#L771

is flagged as a suspicious one by [VaultPress](https://vaultpress.com/) (backup and security site scanning solution by Automattic for WordPress blogging platform).

They are aware that this is a false positive, but is it possible to remove the link from the source code altogether, so it won't be flagged in the future?

Related discussion on wordpress.org forum: https://wordpress.org/support/topic/vaultpress-flags-plugin-with-a-suspicious-link/

**Comment by leonstafford** [comment]
How about escaping the URL in a way that's still readable for a human but won't trip their radar?

**Comment by slaFFik** [comment]
`https://crypto.stackexchange.com/q/5708` is the short form of the link - shortening most likely won't help.

`crypto.stackexchange.com/q/5708` or `crypto.stackexchange.com/questions/5708/creating-a-small-number-from-a-cryptographically-secure-random-string` may help though, still readable, but not clickable.

**Comment by leonstafford** [comment]
More like crypto DOT stackexchange DOT com/q/5708 or such. Still helps a dev reading the code. 

But, still, should push Automattic to fix their false positive, not make silly changes for them ¯\_(ツ)_/¯

**Comment by terrafrost** [comment]
I agree that Automattic should fix it but in the mean time I'm willing to make changes.

Would a bit.ly link be flagged as suspicious? eg. https://bit.ly/2B1H4gG ?

**Comment by slaFFik** [comment]
@terrafrost I've asked the user that initially reported this issue to re-check with VaultPress support. I don't have access to their security scanner.
